client/client_manager.go

```diff
@@ -10,7 +10,9 @@ import (
 )
 
 type ExecutorClientManager interface {
+	MasterClient() MasterClient
```

Comment:
Why is it necessary for an `ExecutorClientManager` to expose a `MasterClient`?

Reply:
The interface name is not accurate, it is a client manager interface actually, which manages both executor clients and master client.

Reply:
Can we change the interface name?

Reply:
Yep, changed it to `ClientsManager`